### ISSUE TITLE
Screenreader-only text

### ISSUE THREAD

**Issue by unindented** [necolas/react-native-web#1181]
**Is your feature request related to a problem? Please describe.**
I want my app to have a nice table of contents for screenreaders, but not all headings should be visible to sighted users (e.g. navigation sections).

**Describe a solution you'd like**
People use variations of this technique on web: https://github.com/twbs/bootstrap/blob/38ca58bf3c886d99e75e307dd6f631a2c9e2c7a2/scss/mixins/_screen-reader.scss#L6-L15

**Describe alternatives you've considered**
I'd like to hear from @necolas what's the most RNW way of doing this.

**Comment by necolas** [comment]
The way to do this is the "web way" at the moment. I've mentioned this use case in a RN accessibility proposal to find out how native does the equivalent https://github.com/react-native-community/discussions-and-proposals/pull/56

**Comment by necolas** [comment]
This is something that should be handled in user-space for now. I won't be adding an API to this library as it won't have any compatibility with React Native
